policy/modules/contrib/iscsi.te

```diff
@@ -120,6 +120,7 @@ modutils_read_module_config(iscsid_t)
 modutils_read_module_deps_files(iscsid_t)
 
 mount_read_pid_files(iscsid_t)
+domain_read_all_domains_state(iscsid_t)
```

Comment:
There is already `domain_dontaudit_read_all_domains_state(iscsid_t)` a few lines above - you should just replace that line with this one. (The `domain_*()` calls should be all together in one block.)

Reply:
Thanks. I have fixed it. This rule was not added before, probably because the failure of priority setting will not affect the function.